test/e2e/csv_e2e_test.go

```diff
@@ -16,7 +16,6 @@ import (
 	rbacv1 "k8s.io/api/rbac/v1"
 	"k8s.io/apiextensions-apiserver/pkg/apis/apiextensions"
```

Comment:
I'm surprised this package wasn't removed - it looks like it's still used to register the sheme:

```
	if err := apiextensions.AddToScheme(scheme); err != nil {
		return nil, err
	}
```

Is that scheme still needed or is fine to keep in place?